src/Tests/Microsoft.Diagnostics.Monitoring.ConfigurationSchema/SchemaGenerator.cs

```diff
@@ -11,29 +12,104 @@ internal sealed class SchemaGenerator
+            schema.Definitions[nameof(ConsoleLoggerOptions)].AnyOf.Add(simpleConsoleLoggerOptionsSchema);
+            schema.Definitions[nameof(ConsoleLoggerOptions)].AnyOf.Add(systemdConsoleLoggerOptionsSchema);
+            schema.Definitions[nameof(ConsoleLoggerOptions)].AnyOf.Add(defaultConsoleLoggerOptionsSchema);
+
             string schemaPayload = schema.ToJson();
 
             //Normalize newlines embedded into json
             schemaPayload = schemaPayload.Replace(@"\r\n", @"\n", StringComparison.Ordinal);
             return schemaPayload;
         }
+
+        public static JsonSchema GenerateConsoleLoggerOptionsSchema(JsonSchema consoleFormatterOptions, ConsoleLoggerFormat consoleLoggerFormat)
+        {
+            JsonSchema consoleLoggerOptionsSchema = new JsonSchema();
+
+            JsonSchemaProperty formatterNameProperty = new JsonSchemaProperty();
+            JsonSchemaProperty formatterOptionsProperty = new JsonSchemaProperty();
+
+            JsonSchema formatterOptionsSchema = new JsonSchema();
+            formatterOptionsSchema.Reference = consoleFormatterOptions;
+
+            formatterOptionsProperty.Reference = formatterOptionsSchema;
+
+            formatterNameProperty.ExtensionData = new Dictionary<string, object>();
+            formatterNameProperty.ExtensionData.Add("const", consoleLoggerFormat.ToString());
+
+            consoleLoggerOptionsSchema.Properties.Add(nameof(ConsoleLoggerOptions.FormatterName), formatterNameProperty);
+            consoleLoggerOptionsSchema.Properties.Add(nameof(ConsoleLoggerOptions.FormatterOptions), formatterOptionsProperty);
+            consoleLoggerOptionsSchema.RequiredProperties.Add(nameof(ConsoleLoggerOptions.FormatterName));
+
+            return consoleLoggerOptionsSchema;
+        }
+
+        public static JsonSchema GenerateDefaultConsoleLoggerOptionsSchema(JsonSchema consoleFormatterOptions)
+        {
+            JsonSchema consoleLoggerOptionsSchema = new JsonSchema();
+
+            JsonSchemaProperty formatterNameProperty = new JsonSchemaProperty();
+            JsonSchemaProperty formatterOptionsProperty = new JsonSchemaProperty();
+            
+            JsonSchema formatterOptionsSchema = new JsonSchema();
+            formatterOptionsSchema.Reference = consoleFormatterOptions;
+
+            formatterOptionsProperty.Reference = formatterOptionsSchema;
+
+            formatterNameProperty.ExtensionData = new Dictionary<string, object>();
+            formatterNameProperty.ExtensionData.Add("const", JsonObjectType.Null);
+
+            consoleLoggerOptionsSchema.Properties.Add(nameof(ConsoleLoggerOptions.FormatterName), formatterNameProperty);
+            consoleLoggerOptionsSchema.Properties.Add(nameof(ConsoleLoggerOptions.FormatterOptions), formatterOptionsProperty);
+            consoleLoggerOptionsSchema.Properties[nameof(ConsoleLoggerOptions.FormatterName)].Default = "Simple";
```

Comment:
You already have access to this via the `formatterNameProperty` variable, so do this instead:
```
formatterNameProperty.Default = "Simple";
```